source/beamline/BioXAS/BioXASSideBeamline.h

```diff
@@ -35,6 +35,8 @@ along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
 #include "beamline/CLS/CLSBasicCompositeScalerChannelDetector.h"
 #include "beamline/CLS/CLSMAXvMotor.h"
 #include "beamline/CLS/CLSJJSlit.h"
+#include "beamline/CLS/CLSJJSlitGapControl.h"
+#include "beamline/CLS/CLSJJSlitCenterControl.h"
```

Comment:
Technically, no.  The JJSlits class contains both JJ slits but a specific JJSlitGapControl only controls a single gap.  Grammar wise, it is okay this way.